lib/lighthouse/benefits_claims/constants.rb

```diff
@@ -5,5 +5,80 @@ module Constants
     CLAIM_TYPE_LANGUAGE_MAP = {
       'Death' => 'expenses related to death or burial'
     }.freeze
+
+    FRIENDLY_DISPLAY_MAPPING_FIRST_PARTY = {
```

Comment:
👏 thanks for moving this over - I think it'll make it cleaner to manage as more get added 😄